CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # Node.js Agent for Google Cloud Trace ChangeLog
 
+## 2017-05-15, Version 2.0.0 (Experimental), @matthewloring
+
+### Notable changes
+
+This release drops support for versions of Node.js <4.
+
+**Support**
```

Comment:
Given that this is a major release, it would be good to call this headline 'semver-major'. That's the main thing people will be looking for in the changelog. I am not sure 'support' means much to people anyway.